### PR TITLE
frontend: Installation Complete screen assets download and UI changes

### PR DESCRIPTION
**Pull request by kyoto**
Partially implements #363.
  - Add assets download area
  - Text changes

The "Install kubectl" and "Deploy Your First Application" sections will be removed in a separate PR after that info has been moved to Console.

screenshot-1

### CODE REVIEW

**Comment by kyoto** [comment]
ok to test

**Comment by alexsomesan** [comment]
cc: @kans @ggreer

**Comment by kyoto** [comment]
RFR @tlwu2013 @squat

File: installer/frontend/components/success.jsx [hunk 1]
```diff
@@ -32,10 +31,25 @@ export const Success = connect(stateToProps)(
       <a href={`https://${tectonicConsole}`} target="_blank">
         <button className="btn btn-primary wiz-giant-button"
                 style={{marginTop: 20}}
-                onClick={() => handleAllDone(platformType)}>Go to my Tectonic Console&nbsp;&nbsp;<i className="fa fa-external-link"></i></button>
+                onClick={() => handleAllDone(platformType)}>Go to my Tectonic Console</button>
```
Comment (ggreer) [inline installer/frontend/components/success.jsx]: Clicking this opens a new window. Don't we want to keep the icon so the user knows what to expect?

**Comment by kyoto** [comment]
@tlwu2013 Could you answer Geoff's questions regarding the UX choices?

**Comment by tlwu2013** [comment]
Hi @kyoto , @ggreer , 

From our user testings, test participants mostly clicked around the side-nav when they were trying to go back. 

But I also think having a "Back" button at the bottom left is no harm (plus it's an existing pattern within installer too). So chat with more folks and update our final mock here.

I am still working on the Console "First time use" page. So I personally prefer we can wait until the Console changes are ready, and then merge this installer PR at the same time.

Thanks!

**Comment by ggreer** [comment]
OK. I don't like long-lived PRs, so I think the best thing to do here would be to put the back button back and merge it.

**Comment by kyoto** [comment]
@ggreer @tlwu2013 OK, I've reduced this PR so that it no longer removes the back link or button icon and I've updated the description / screenshot.

**Pull request merged**